TesseractCore/Importing.swift

```diff
@@ -0,0 +1,62 @@
+//  Copyright (c) 2015 Rob Rix. All rights reserved.
+
+private func nub<T: Equatable>(list: [T]) -> [T] {
```

Comment:
Oh, this is straight outta Haskell: http://hackage.haskell.org/package/base-4.7.0.2/docs/Data-List.html#g:20

That name is pretty confusing, so I'll just change it to `unique` or something.